examples/quickstart/00-rbac.jsonnet

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 Heptio Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+local k = import "ksonnet.beta.2/k.libsonnet";
+
+local conf = {
+    namespace: "heptio-sonobuoy",
+    labels: {
+        component: "sonobuoy",
+    },
+    serviceAccount: {
+        name: "sonobuoy-serviceaccount",
+    },
+
+    # shared metadata
+    metadata: {
+        name: $.serviceAccount.name,
+        labels: $.labels,
+    },
+
+};
+
+local namespace = local ns = k.core.v1.namespace;
```

Comment:
Super tiny micro nit, when `jsonnet fmt` matures, this local will get indented below. AFAIK that's actively being worked on by Dave's intern.